agent/src/main/java/com/code_intelligence/jazzer/runtime/TraceCmpHooks.java

```diff
@@ -301,6 +303,19 @@ public static void mapGet(
         // compared to the maps keys.
       }
     } else if (currentKey instanceof Comparable) {
+      if (map instanceof LinkedHashMap) {
```

Comment:
Is this specific to `LinkedHashMap` or are there possibly other implementations with this behavior? Should we wrap the entire hook into a try...catch block?